src/test/java/io/zold/api/WalletTest.java

```diff
@@ -45,7 +46,10 @@
  * @checkstyle JavadocMethodCheck (500 lines)
  * @checkstyle JavadocVariableCheck (500 lines)
  * @checkstyle MagicNumberCheck (500 lines)
+ * @checkstyle LineLengthCheck (500 lines)
```

Comment:
@Vatavuk let's suppress `LineLength` on a case-by-case basis, in order to prevent abuse